test/elixir/test/changes_async_test.exs

```diff
@@ -51,13 +50,13 @@ defmodule ChangesAsyncTest do
 
     create_doc_bar(db_name, "bar")
 
-    {changes_length, last_seq_prefix} =
+    changes =
       req_id.id
       |> process_response(&parse_chunk/1)
-      |> parse_changes_response()
 
-    assert changes_length == 1, "should return one change"
-    assert last_seq_prefix == "2-", "seq must start with 2-"
+    assert length(changes["results"]) == 1, "should return one change"
+    next_last_seq = changes["last_seq"]
+    assert next_last_seq > last_seq
```

Comment:
Now seqs are comparable?

Reply:
I think they were comparable previously, too. Now the only thing about them is that they are monotonously increasing.
+CC @kocolosk